.changeset/smooth-doors-fly.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sveltejs/kit': patch
+'create-svelte': patch
+---
+
+Upgrade to Vite 4
```

Comment:
isn't this also `[breaking]` because it's a peer dependency for existing projects?

Reply:
done